.github/workflows/ci.yml

```diff
@@ -6,106 +6,98 @@ on:
       - master
       - develop
       - "release/**"
-env:
-  APPLITOOLS_API_KEY: ${{ secrets.APPLITOOLS_API_KEY }}
-  APPLITOOLS_BATCH_ID: ${{ github.event.pull_request.head.sha }}
 jobs:
-  prepare:
-    runs-on: ubuntu-latest
-    outputs:
-      deploy: ${{ steps.filter.outputs.deploy }}
-      vrt: ${{ steps.filter.outputs.vrt }}
-    steps:
-      - uses: dorny/paths-filter@v2
-        id: filter
-        with:
-          filters: |
-            deploy:
-              - './packages/odyssey-react/**'
-              - './packages/odyssey/src/**'
-            vrt:
-              - './packages/odyssey-react/src/**'
-              - './packages/odyssey/src/**'
```

Comment:
Sorry, I recognize that wasnt clear.

At the bottom of the PR screen it tells you which jobs passed and which jobs failed. By having one job, its not clear which part of the job failed. Thats OK though, since the details are in the run.